Lib/test/test_locale.py

```diff
@@ -583,6 +584,30 @@ def test_invalid_iterable_in_localetuple(self):
             locale.setlocale(locale.LC_ALL, (b'not', b'valid'))
 
 
+class TestGetFirstWeekDay(BaseLocalizedTest):
+
+    locale_type = locale.LC_TIME
+
+    def test_getfirstweekday(self):
+        if platform.libc_ver()[0] == 'glibc':
+            self.assertEqual(locale.getfirstweekday(), 0)
+        elif platform.system() == 'Windows':
+            self.assertEqual(locale.getfirstweekday(), 6)
+        else:
+            self.assertEqual(locale.getfirstweekday(), None)
```

Comment:
Have you verified that the test passes on musl libc?

Reply:
I do not have access to such system.